packages/suricata.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,114 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+Import-Module powershell-yaml
+
+# set configurations
+$toolName = "Suricata"
+$category = VM-Get-Category($MyInvocation.MyCommand.Definition)
+$filetype = "MSI"
+$toolDir = Join-Path ${Env:ProgramFiles} $toolName
+$executablePath = Join-Path $toolDir "suricata.exe"
+$url = "https://www.openinfosecfoundation.org/download/windows/Suricata-7.0.8-1-64bit.msi"
+$sha256 = "8bdd78d2978e4efc6d23ab1ced024342cac7d38afb152a6e3f70ac5182bd8cd4"
+$silentArgs = "/qn /norestart"
+
+$packageArgs = @{
+    toolName = $toolName
+    category = $category
+    filetype = $filetype
+    silentArgs = $silentArgs
+    executablePath = $executablePath
+    url = $url
+    sha256 = $sha256
+    consoleApp = $true
+}
+
+# download msi file
+VM-Install-With-Installer @packageArgs
+VM-Assert-Path $executablePath
+
+# delete default desktop shortcut
+try{
+    $desktopShortcutPath = "${Env:HomeDrive}\Users\*\Desktop\$toolName*.lnk"
+    Remove-Item -Path $desktopShortcutPath -ErrorAction SilentlyContinue
+}
+catch{
+    VM-Write-Log-Exception $_
+}
+
+# rules configuration and download
+$rulesXmlPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)/rules.xml"
+$rulesXml = [xml](Get-Content $rulesXmlPath)
+
+$rulesDir = Join-Path $toolDir "rules" -Resolve
+$rulesConfigPath = Join-Path $toolDir "suricata.yaml" -Resolve
+
+$rulesConfig = ConvertFrom-Yaml (Get-Content -Raw -Path $rulesConfigPath)
```

Comment:
I would move the definition of these two variables just before they are used (before the code to add rules to `suricata.yaml`)